React/Fabric/Mounting/ComponentViews/ScrollView/RCTEnhancedScrollView.h

```diff
@@ -27,7 +27,7 @@ NS_ASSUME_NONNULL_BEGIN
  * `UIScrollView` subclass which has some improvements and tweaks
  * which are not directly related to React Native.
  */
-@interface RCTEnhancedScrollView : UIScrollView
+@interface RCTEnhancedScrollView : RCTUIScrollView // TODO(macOS GH#774)
```

Comment:
Just an FYI, I think `<ScrollView>` is one of the Paper controls on macOS with a lot of custom logic to handle scrollbars and yoga layout for them. I expect this file will be hit more in the future :)